stac_ipyleaflet/core.py

```diff
@@ -113,7 +117,7 @@ def __init__(self, **kwargs):
         self.add_custom_tools()
         self.draw_control = DrawControlWidget.template(self)
 
-        return None
```

Comment:
📓 : by default I think python already returns none

Reply:
That's true - even though I (personally) think it's better to be explicit about it, it's basically a style difference. What's more important is that we're consistent - if possible, in our general approach, but more importantly within a function.

From PEP8:
_Be consistent in return statements. Either all return statements in a function should return an expression, or none of them should. If any return statement returns an expression, any return statements where no value is returned should explicitly state this as return None, and an explicit return statement should be present at the end of the function (if reachable):_

https://peps.python.org/pep-0008/#programming-recommendations

Reply:
👍🏼 to being explicit. Can add back